packages/frontend/editor-ui/src/stores/nodeCreator.store.ts

```diff
@@ -181,11 +181,9 @@ export const useNodeCreatorStore = defineStore(STORES.NODE_CREATOR, () => {
 			return;
 		}
 
-		const { type, index, mode } = parseCanvasConnectionHandleString(connection.sourceHandle);
+		const { type, mode } = parseCanvasConnectionHandleString(connection.sourceHandle);
 
 		uiStore.lastSelectedNode = sourceNode.name;
-		uiStore.lastSelectedNodeEndpointUuid = connection.sourceHandle ?? null;
-		uiStore.lastSelectedNodeOutputIndex = index;
```

Comment:
Values are assigned to these two fields but are referenced from nowhere. Do you think it's intentional? @alexgrozav

Reply:
It's leftover code from before the node view rewrite. Should be safe to delete